src/components/button/Button.tsx

```diff
@@ -1,21 +1,24 @@
 import * as React from 'react';
-import { View, Text, TouchableOpacity, TouchableNativeFeedback, GestureResponderEvent, Platform, AccessibilityTrait, ViewStyle } from 'react-native';
-
-const s = require('./Button.css');
+import { View, Text, TouchableOpacity, TouchableNativeFeedback, GestureResponderEvent,
+  Platform, AccessibilityTrait, ViewStyle, TextStyle } from 'react-native';
+import * as s from './Button.css';
 
 interface IProps {
   title: string;
+  testID?: string;
   accessibilityLabel?: string;
   disabled?: boolean;
   style?: ViewStyle;
   hasTVPreferredFocus?: boolean;
-  onPress?: (event: GestureResponderEvent) => void;
-  testID?: string;
+  onPress?(event: GestureResponderEvent): void;
 }
 
-export default class Button extends React.PureComponent<IProps> {
+// tslint:disable-next-line variable-name typedef
```

Comment:
- I would pass `allow-pascal-case` for the variable-name rule because we use it a lot. For example with `componentType` when checking if children are equal to a specific type of component.
- I don't know if `typedef` is not too strict